x-pack/solutions/security/plugins/security_solution/server/request_context_factory.ts

```diff
@@ -21,7 +21,7 @@ import { AssetInventoryDataClient } from './lib/asset_inventory/asset_inventory_
 import { createDetectionRulesClient } from './lib/detection_engine/rule_management/logic/detection_rules_client/detection_rules_client';
 import type { IRuleMonitoringService } from './lib/detection_engine/rule_monitoring';
 import { AssetCriticalityDataClient } from './lib/entity_analytics/asset_criticality';
-import { getApiKeyManager } from './lib/entity_analytics/entity_store/auth/api_key';
+import { getApiKeyManager as buildApiKeyManager } from './lib/entity_analytics/entity_store/auth/api_key';
```

Comment:
Makes me think this function should just be renamed? I'd prefer to avoid using `as`

Reply:
I only renamed it because of a naming conflict that no longer exists. I will roll back the change.